tests/integ/scala/test_dataframe_suite.py

```diff
@@ -2548,14 +2573,15 @@ def test_fillna(session):
     )
     # wrong column name
     with pytest.raises(SnowparkColumnException) as ex_info:
-        TestData.null_data3(session).na.fill({"wrong": 11})
+        TestData.null_data3(session, local_testing_mode).na.fill({"wrong": 11})
     assert "The DataFrame does not contain the column named" in str(ex_info)
 
 
-def test_replace(session):
+@pytest.mark.localtest
+def test_replace(session, local_testing_mode):
     res = (
-        TestData.null_data3(session)
-        .na.replace({2: 300, 1: 200}, subset=["flo"])
+        TestData.null_data3(session, local_testing_mode)
+        .na.replace({2: 300.0, 1: 200.0}, subset=["flo"])
```

Comment:
does it mean `300` and `200` don't work in local test?

Reply:
Because `300` and `200` are integer values, we need coercion support to use them on a floating point column.